packages/dashboard/src/pages/Wallets/components/Transfer/TransferERC721.tsx

```diff
@@ -171,6 +171,7 @@ export const TransferERC721 = memo(() => {
     const handleSelectNFT = async () => {
         const contract = await selectNFTContract({
             pluginID: NetworkPluginID.PLUGIN_EVM,
+            isERC721Only: true,
```

Comment:
You can add `schemaType` in `HubOptions_Base`.

Reply:
ok